src/styles/styles.ts

```diff
@@ -20,7 +20,7 @@ import overflowXHidden from './overflowXHidden';
 import pointerEventsAuto from './pointerEventsAuto';
 import pointerEventsBoxNone from './pointerEventsBoxNone';
 import pointerEventsNone from './pointerEventsNone';
-import defaultTheme from './themes/default';
+import {defaultTheme} from './themes/Themes';
```

Comment:
is this meant to be caps

Reply:
for now yes, we can rename and re-structure this in another PR